### PR TITLE
Buy ADA button

### PR DESCRIPTION
**Pull request by ladomyriak**

### CODE REVIEW

File: app/public/css/styles.css [hunk 1]
```diff
@@ -21,6 +21,7 @@
   --color-pink: #e72076;
   --color-grey-dark: #20323f;
   --color-grey: #606a71;
+  --color-grey-medium: rgba(96,106,113, .6);
```
Comment (d-novak) [inline app/public/css/styles.css]: Let's have this as `hex` like the rest of the colors, `#99A0A4 `.

File: app/public/css/styles.css [hunk 2]
```diff
@@ -1646,6 +1647,29 @@ button[data-balloon] {
   margin-right: 16px;
 }
 
+.buy-ada {
+  margin-top: 40px;
+  display: flex;
+  align-items: center;
+}
+
+.buy-ada-link,
```
Comment (d-novak) [inline app/public/css/styles.css]: Extract this to a separate class that can be used on any button, ie. `.button.link`

File: app/frontend/components/common/balance.js [hunk 3]
```diff
@@ -40,7 +41,32 @@ const Balance = ({balance, reloadWalletInfo, conversionRates}) =>
...
+      ),
+      h(
+        'span',
+        {
+          class: 'buy-ada-logo',
+        },
+        'Powered by',
+        h('img', {
+          class: 'buy-ada-icon',
+          src: LOGO_COINSWITCH,
```
Comment (refi93) [inline app/frontend/components/common/balance.js]: I would directly reference the asset url here. Other parts of the code reference the assets directly as well

File: app/public/css/styles.css [hunk 4]
```diff
@@ -1646,6 +1647,29 @@ button[data-balloon] {
...
+.buy-ada-link:hover {
+  text-decoration: none;
+}
+
+.buy-ada-logo {
```
Comment (refi93) [inline app/public/css/styles.css]: I would name this `buy-ada-partner-logo`

File: app/frontend/components/common/balance.js (hunk 3 above)
Comment (d-novak) [reply app/frontend/components/common/balance.js]: @refi93 Thanks, we discussed this in person with @ladomyriak  👍

File: app/public/css/styles.css [hunk 5]
```diff
@@ -1646,6 +1647,29 @@ button[data-balloon] {
...
+  margin-left: 24px;
+  font-size: 14px;
+  color: var(--color-grey-medium);
+  padding-bottom: 4px;
+}
+
+.buy-ada-icon {
```
Comment (refi93) [inline app/public/css/styles.css]: not sure what are good css naming conventions but this seems like it should be named `buy-ada-partner-logo` and the wrapping element (the one named `buy-ada-logo` now if I understand right) could be named `buy-ada-partner-logo-wrapper`

File: app/public/css/styles.css (hunk 1 above)
Comment (ladomyriak) [reply app/public/css/styles.css]: Fixed

File: app/public/css/styles.css (hunk 2 above)
Comment (ladomyriak) [reply app/public/css/styles.css]: Fixed

File: app/frontend/components/common/balance.js (hunk 3 above)
Comment (ladomyriak) [reply app/frontend/components/common/balance.js]: Fixed

File: app/public/css/styles.css (hunk 5 above)
Comment (ladomyriak) [reply app/public/css/styles.css]: okay, use `buy-ada-partner` as the main class name, then `buy-ada-partner-wrapper` etc.

File: app/frontend/components/common/balance.js [hunk 6]
```diff
@@ -40,7 +40,32 @@ const Balance = ({balance, reloadWalletInfo, conversionRates}) =>
...
+        'Buy ADA'
+      ),
+      h(
+        'span',
+        {
+          class: 'buy-ada-partner-logo',
+        },
+        'Powered by',
+        h('img', {
+          class: 'buy-ada-partner-icon',
```
Comment (refi93) [inline app/frontend/components/common/balance.js]: I would rather name this `buy-ada-partner-logo-img` having both "logo" and "icon" in the same context seems a bit confusing to me

File: app/frontend/components/common/balance.js [hunk 7]
```diff
@@ -40,7 +40,32 @@ const Balance = ({balance, reloadWalletInfo, conversionRates}) =>
...
+          src: 'assets/coinswitch-logo.svg',
+          alt: 'COINSWITCH logo',
```
Comment (refi93) [inline app/frontend/components/common/balance.js]: COINSWITCH -> CoinSwitch (that's how they have it on their web)

**Pull request merged**